style.css

```diff
@@ -1 +1,129 @@
 /*  Add your styling here */
+
+body{
+    text-align: center;
+    background-color: rgb(116, 72, 72);
+}
+
+.head-bar{
+    display:flex;
+    justify-content: space-between ;
+
+}
+
+.img-logo{
+    width: 20%;
+    height: auto;
+}
+
+.hamburger {
+    display: inline-block;
+}
+
+/* hero section starts here */
+.welcomeHero {
+    display: flex;
+    text-align: justify;
+    text-justify: inter-word;
+
+
+}
+
+.cakeDisp img {
+display: inline-block;
+/* flex-direction: row; */
+width: 45%;
+height:;
+
+justify-content: space-between;
+padding: 5px;
+}
+
+.imgChocoBig {
+flex:auto;
+width: 100%;
+height: ;
+
+}
+/* hero section ends here*/
+
+
+
+/* img {
+    padding: 10px; */
+    /* list-style-type: none; */
+}
+
+
+
+.ulNav li a{
+    list-style-type: none;
+    display: flex;
+    
+    
+}
+ul li{
+    display: inline-flex;
+    margin: 10px;
+    list-style: none;
+    flex-direction: row;
+}
+nav {
+    display: none;
+    }
+
+
+ h1 {
+    font-family: Verdana, Tahoma, sans-serif;
+    }
+
+ul.social li a img{
+    width:50px;
+    height:50px;
+    
+}
+
+.socialOrd {
+    display:flex;
+    padding: 0;
+    justify-content: space-around;
+}
+
+/* media query */
+@media (min-width: 400px) {
```

Comment:
Good work using 2 breakpoints. Since the smallest phone is 320px wide, it's best for your first media query to start a little bigger, for example 550px or 600px.